tests/CollectionDiffTest.php

```diff
@@ -48,78 +63,52 @@ public function testCompareWithCompare()
                 "price" => 8.5,
             ],
         ];
+    }
 
-        $serializer = new Serializer([new PropertyNormalizer()]);
-
-        $collectionUpdater = new CollectionDiff($serializer, ['name', 'category'], $from, $to, ['defaultCompare' => 'compare']);
+    public function testCompareWithOldValues()
+    {
+        $this->collectionDiff->setAddOldValues(true)->compare(['name', 'category'], $this->from, $this->to);
```

Comment:
comme il y a plusieurs appels sur une ligne, c'est plus parlant un appel par ligne :
```suggestion
        $this
            ->collectionDiff
            ->setAddOldValues(true)
            ->compare(['name', 'category'], $this->from, $this->to);
```